src/components/slide-toggle/slide-toggle.scss

```diff
@@ -34,21 +48,24 @@ $md-slide-toggle-margin: 16px !default;
   outline: none;
 
   &.md-checked {
-    @include md-switch-checked($md-accent);
-
-    &.md-primary {
-      @include md-switch-checked($md-primary);
-    }
-
-    &.md-warn {
-      @include md-switch-checked($md-warn);
-    }
-
     .md-slide-toggle-thumb-container {
       transform: translate3d(100%, 0, 0);
     }
   }
 
+  @include md-switch-checked($md-accent);
+  @include md-switch-ripple($md-accent);
+
+  &.md-primary {
+    @include md-switch-checked($md-primary);
+    @include md-switch-ripple($md-primary);

```

Comment:
FYI: I knew, this could be grouped up, into one mixin. But since this is quite temporary more more clear, I decided to do it this way.